application/src/test/java/org/thingsboard/server/transport/coap/attributes/updates/CoapAttributesUpdatesIntegrationTest.java

```diff
@@ -59,11 +60,13 @@ public void afterTest() throws Exception {
         processAfterTest();
     }
 
+    @Ignore
```

Comment:
@Ignore // Uncomment when Californium 3.11 is released with [PR 2215](https://github.com/eclipse-californium/californium/pull/2215)